src/main/java/org/hibernate/infra/replicate/jira/service/jira/JiraService.java

```diff
@@ -307,12 +312,40 @@ public void acknowledge(String project, JiraWebHookEvent event) {
 				throw new ConstraintViolationException("Project " + project + " is not configured.", Set.of());
 			}
 
+			if (context.isUserIgnored(triggeredByUser)) {
+				Log.infof("Event was triggered by %s user that is in the ignore list.", triggeredByUser);
```

Comment:
```suggestion
				Log.tracef("Event was triggered by %s user that is in the ignore list.", triggeredByUser);
```

Unless you want to be flooded, of course :]